src/test_l1_tvSettings.c

```diff
@@ -345,7 +311,7 @@ void test_l1_tvSettings_positive_RegisterVideoFormatChangeCB (void)
 	callbackData.cb = tvVideoFormatChangeHandler;
 
 	result = RegisterVideoFormatChangeCB(&callbackData);
-	UT_ASSERT_AUTO_TERM_NUMERICAL(result, tvERROR_NONE);
+	UT_ASSERT_EQUAL(result, tvERROR_NONE);
```

Comment:
Line no 300 - Initialize the structure to 0

Reply:
Added initialization to all the places this comment applies.